src/main/java/millom/sandbox/pojos/Weather.java

```diff
@@ -0,0 +1,41 @@
+package millom.sandbox.pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Weather {
+  @JsonProperty("descriptions")
+  private Descriptions descriptions;
+  @JsonProperty("soles")
+  private List<Sol> soles;
+  private Weather(){
+  }
+
+  private Weather(WeatherBuilder weatherBuilder){
+    this.descriptions = weatherBuilder.descriptions;
+    this.soles = weatherBuilder.soles;
+  }
+
+  public Descriptions getDescriptions() {
+    return descriptions;
+  }
+
+  public List<Sol> getSoles() {
+    return soles;
+  }
+
+  public static class WeatherBuilder {
```

Comment:
That's not quite the builder pattern. Builders return themselves until the final stage — `.build()` when they return the target object. More on this here: https://howtodoinjava.com/design-patterns/creational/builder-pattern-in-java/

For the `Weather`, a bulder implementation could be 
```
package millom.sandbox.pojos;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;
@JsonIgnoreProperties(ignoreUnknown = true)
public class Weather {
  private Descriptions descriptions;
  private List<Sol> sols;

  public Weather() {}

  private Weather(Descriptions descriptions, List<Sol> sols){
    this.descriptions = descriptions;
    this.sols = sols;
  }

  public Descriptions getDescriptions() {
    return descriptions;
  }

  public List<Sol> getSols() {
    return sols;
  }

  public static WeatherBuilder builder() {
    return new WeatherBuilder();
  }

  public static class WeatherBuilder {
    
    private Descriptions descriptions;
    private List<Sol> sols;

    public WeatherBuilder descriptions(Descriptions descriptions) {
      this.descriptions = descriptions;
      return this;
    }

    public WeatherBuilder sols(List<Sol> sols) {
      this.sols = sols;
      return this;
    }
    
    public Weather build() {
      return new Weather(descriptions, sols);
    }
    
  }

}
```

And then you can use it like this:
```
public static final Weather WEATHER_DATA = Weather.builder().sols(SOLS).descriptions(DESCRIPTIONS).build();
```

Reply:
Implemented builder pattern as mentioned above for Descriptions, Sol and Weather classes.

In the above code instead of creating builder() returns WeatherBuilder, created new Weather.WeatherBuilder().

Removed ```
public static WeatherBuilder builder() {
    return new WeatherBuilder();
  }

```
And changed to 
```
public static final Weather WEATHER_DATA = new Weather.WeatherBuilder().sols(SOLS)
      .descriptions(DESCRIPTIONS).build();
```